src/main/resources/dbmigration/V1__USER_ACCOUNT_TABLE.sql

```diff
@@ -0,0 +1,10 @@
+create table USER_ACCOUNT
```

Comment:
please move this sql script to V1_00__INITIALIZE.sql
and add constraints to user_id
and delete suffix table from file name